### ISSUE TITLE
Failed to upgrade legacy queries

### ISSUE THREAD

**Issue by DifferentialOrange** [tarantool/grafana-dashboard#186]
@opomuc reported that installing Tarantool dashboard on Grafana 9.2.2 failed with

```
Failed to upgrade legacy queries Datasource ${DS_PROMETHEUS} was not found
```

but I failed to reproduce this one.

**Comment by opomuc** [comment]
This happens when using Dashboard provisioning and creating dashboards from files on disk.

**Comment by DifferentialOrange** [comment]
The reason why this error was raised is that the dashboard was imported through Grafana provisioning: exporting a raw json fails since there is no way to fill `__inputs` import datasource variable.

It seems that reworking import variables to be similar to what used in node exported dashboard (see #185) should solve this one. It is also may be helpful to rework datasource later (not until we deprecate 8.x), see #189 .

For now, this ticket is closed in favor of #185 . The solution which will close this one is expected to solve provisioning import too.
